core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -332,4 +345,25 @@ ListenableFuture<InstanceData> getCurrent() {
   public CloudSqlInstanceName getInstanceName() {
     return instanceName;
   }
+
+  public void close() {
```

Comment:
Should this be public? Shouldn't the close be exposed on the CoreSocketFactory instead?

Reply:
Fixed. This should be package scope.